go.mod

```
@@ -0,0 +1,5 @@
+module github.com/linesmerrill/go-mux-template
```

As this will be inside of our project, you will need to update the module

```suggestion
module github.com/webgoprojects/go-mux-template
```

---

Oh snap! I missed that, my bad. Fixing now.